CONTRIBUTING.md

```diff
@@ -59,14 +59,23 @@ npm test
 
 ### Integration Tests
 
+From the top-level xrpl.js folder (one level above `packages`), run the following commands:
+
 ```bash
 npm install
 # sets up the rippled standalone Docker container - you can skip this step if you already have it set up
-docker run -p 6006:6006 -it natenichols/rippled-standalone:latest
+docker run -p 6006:6006 -it -v $PWD/.ci-config:/config/ xrpllabsofficial/xrpld:latest -a --start
```

Comment:
I recommend using the expanded versions of commands in the docs too (`--volume` makes more sense than `-v`)